pyeudiw/satosa/backend.py

```diff
@@ -230,20 +229,18 @@ def _translate_response(self, response, issuer):
         return internal_resp
 
     def _handle_vp(self, vp_token: str, context: Context) -> dict:
-        valid, value = check_vp_token(
-            vp_token, self.config, self.sd_specification, self.sd_jwt)
+        valid, value = None, None
+        
+        try:
+            valid, value = check_vp_token(
+                vp_token, None, self.metadata_jwks_by_kids)
+        except Exception as e:
```

Comment:
here the exception should be specialized for this case, using the already defined exception InvalidVPToken

Reply:
Resolved in [6f6ce24](https://github.com/italia/eudi-wallet-it-python/pull/89/commits/6f6ce24fe06dbf4c63d54a8ae9e2b2090a8f65f5)